src/applications/edu-benefits/complaint-tool/config/form.js

```diff
@@ -27,18 +65,183 @@ const formConfig = {
+                  Myself: () => <div className="usa-alert-info no-background-image"><i>(We’ll only share your name with the school.)</i></div>,
+                  'Someone else': () => <div className="usa-alert-info no-background-image"><i>(We’ll only share your name with the school.)</i></div>,
+                  'I want to submit my feedback anonymously': () => <div className="usa-alert-info no-background-image"><i>(Your personal information won’t be shared with anyone outside of VA.)</i></div>
+                },
+                expandUnderClassNames: 'schemaform-expandUnder',
+              }
+            },
+            fullName: _.merge(fullNameUI, {
+              prefix: {
+                'ui:title': 'Prefix',
+                'ui:options': {
+                  widgetClassNames: 'form-select-medium'
+                }
+              },
+              first: {
+                'ui:title': 'Your first name',
+                'ui:required': (formData) => !!formData && (formData.onBehalfOf !== anonymous)
+              },
+              last: {
+                'ui:title': 'Your last name',
+                'ui:required': (formData) => !!formData && (formData.onBehalfOf !== anonymous)
+              },
+              middle: {
+                'ui:title': 'Your middle name'
+              },
+              suffix: {
+                'ui:title': 'Your suffix'
+              },
+              'ui:options': {
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: isNotAnonymous
+              }
+            }),
+            dob: {
+              'ui:title': 'Date of birth',
+              'ui:options': {
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: isNotAnonymous
+              }
+            },
+            serviceAffiliation: { // could wrap service info in an object
+              'ui:title': 'Service affiliation',
+              'ui:options': {
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: myself
+              },
+              'ui:required': (formData) => !!formData && (formData.onBehalfOf === myself)
+            },
+            serviceBranch: {
+              'ui:title': 'Branch of service',
+              'ui:options': {
+                hideIf: (formData) => {
+                  if (!formData.serviceAffiliation || ((formData.serviceAffiliation !== 'Veteran') && (formData.serviceAffiliation !== 'Service Member'))) {
+                    return true;
+                  }
+                  return false;
+                },
+                expandUnder: 'onBehalfOf',
+                expandUnderCondition: myself
+              }
+            },
+            serviceDateRange: _.merge(dateRangeUI(
+              'Service start date',
+              'Service end date',
+              'End of service must be after start of service'
+            ),
+            {
+              'ui:options': {
+                hideIf: (formData) => {
```

Comment:
Can this be moved into a helper similar to `isNotAnonymous`?

Reply:
Good idea @erikphansen, updated.